libs/mark-flow-ui/src/hooks/use_session_settings_manager.ts

```diff
@@ -49,7 +46,13 @@ export function useSessionSettingsManager(
     previousAuthStatusRef.current &&
     isCardlessVoterAuth(previousAuthStatusRef.current);
   const isLoggedInAsVoter = isCardlessVoterAuth(authStatus);
-  const isVotingSessionActive = !!votes;
+  // If the pollworker deactivated the voter session and initiated a new one, the session expires at timestamp will have changed.
```

Comment:
I think this comment should read "the session IDs will have changed" - probably a relic from an earlier iteration of this code